Added Objects, partial completion with no documentation

garyjlin opened the pull request.

garyjlin committed:
Added Objects, partial completion with no documentation

YuFuTotG commented on Objects/Main.java:
```diff
@@ -0,0 +1,15 @@
+package Objects;
+import Objects.*;
+
+public class Main {
```
Why is there a Main.java
I don't understand the purpose of this. is it necessary?

garyjlin replied:
Nah that was for Objects testing, I've removed it in the latest commit.

YuFuTotG commented:
There is only one commit here. Please commit more frequently.
I have made a note above, please comment on it when you can.

YuFuTotG commented:
Also please include a description.

garyjlin committed:
Merge branch 'master' of https://github.com/YuFuTotG/Summer-Project into GaryLin

garyjlin committed:
Moved Objects from outside Polarity to Polarity/app/src/Objects
Implementation is approximately 80% complete, documentation to be
completed on later push
